controller/WFCORE-4868_configuration-files-aliases.adoc

```diff
@@ -0,0 +1,100 @@
+---
+categories:
+ - controller
+---
+= [Community] Aliases for standard configuration files
+:author:            Michal Petrov
+:email:             mpetrov@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Introduce aliases for standard configuration files
+
+This will allow users to reference the configuration by the suffix instead of the whole filename, e.g. `standalone.sh -c full` instead of `standalone.sh -c standalone-full.xml`
+The alias "abc" is shorthand for <baseName>-abc.xml where "baseName" is the basic name of a configuration file, i.e. "standalone" by default.
+
+In addition some of the basic configuration files could be referenced by an alias.
+
+.Full list of aliases:
+|===
+|  Configuration file | Alias
+
+| *-full-ha.xml          | fha
+
+| *-load-balancer.xml    | lb
+
+| *-microprofile.xml     | mp
+
+| *-microprofile-ha.xml  | mpha
+|===
+
+Special values for `-c` remain in effect, e.g. `-c boot` references the standalone.boot.xml file (if present) and it is not possible to reference a custom file named `standalone-boot.xml` this way.
```

Comment:
I am more inclined to keep the `boot` (and `initial`, v* and so on) untouched as they are. They should not be treated as possible aliases. If I use `-c boot`, this `boot` should not be treated as an alias, it should have the same meaning as before.

Reply:
That's why I'm specifying that `boot` _cannot_ be used as an alias for a file named "standalone-boot.xml"

Reply:
Ahh, got it @michpetrov , I wrongly read this statement, thanks!